patches/0001-batman-adv-fix-when-vlan-crc-0x00000000.patch

```diff
@@ -55,6 +49,3 @@ index 00000000..7d92a1c2
 ++-- 
```

Comment:
This looks odd, and should have been gone.
But maybe this is yet another level, I just have not cleaned up.